### ISSUE TITLE
Role fails with error: dict object has no attribute rc

### ISSUE THREAD

**Issue by TAlonglong** [metno/ansible-role-chrony#22]
Hi!

I've read `CODE_OF_CONDUCT.md` in this repository, and want to report a [bug/issue].

[Description]

The following playbook:
```
    - role: chrony
      become_user: root
      chrony_disable_ntpd: true
      chrony_enable: true
      chrony_log_enable: true
      chrony_log: measurements statistics tracking
      chrony_ntp_servers:
        - <servers>
```

Running the playbook gives me:
```
TASK [chrony : Check if we are running supported os] *****************************************************************************************************************************************************************
ok: [seadas-jammy-a]

TASK [chrony : Chrony variables for ubuntu] **************************************************************************************************************************************************************************
ok: [seadas-jammy-a]

TASK [chrony : Chrony variables for centos] **************************************************************************************************************************************************************************
skipping: [seadas-jammy-a]

TASK [chrony : Install chrony on ubuntu] *****************************************************************************************************************************************************************************
ok: [seadas-jammy-a]

TASK [chrony : Install chrony on centos and redhat] ******************************************************************************************************************************************************************
skipping: [seadas-jammy-a]

TASK [chrony : Check ntpd is running] ********************************************************************************************************************************************************************************
fatal: [seadas-jammy-a]: FAILED! => {"changed": false, "changed_when_result": "The conditional check 'ntpd_running.rc == 0' failed. The error was: error while evaluating conditional (ntpd_running.rc == 0): 'dict object' has no attribute 'rc'. 'dict object' has no attribute 'rc'", "msg": "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: _raw_params, _uses_shell, argv, chdir, creates, executable, removes, stdin, stdin_add_newline, strip_empty_ends."}
```

Is there something I need to change in my setup? Or is it something wrong with the role?

**Comment by KjetilKnutsen** [comment]
Hi @TAlonglong

I have tried to reproduce your error so far, but have not been successful in doing so.

Playbook ran:
```
---
- name: test chrony
  hosts: snapp
  become: true

  roles:
    - role: metno.chrony
      become_user: root
      chrony_disable_ntpd: true
      chrony_enable: true
      chrony_log_enable: true
      chrony_log: measurements statistics tracking
      chrony_ntp_servers:
        - 0.ubuntu.pool.ntp.org
        - 1.ubuntu.pool.ntp.org
        - 2.ubuntu.pool.ntp.org
        - 3.ubuntu.pool.ntp.org
```

Results
```
PLAY [test chrony] ****************************************************************************************************************************************

TASK [Gathering Facts] ************************************************************************************************************************************
ok: [snapp]

TASK [metno.chrony : Check if we are running supported os] ************************************************************************************************
ok: [snapp]

TASK [metno.chrony : Chrony variables for ubuntu] *********************************************************************************************************
ok: [snapp]

TASK [metno.chrony : Chrony variables for centos] *********************************************************************************************************
skipping: [snapp]

TASK [metno.chrony : Install chrony on ubuntu] ************************************************************************************************************
changed: [snapp]

TASK [metno.chrony : Install chrony on centos and redhat] *************************************************************************************************
skipping: [snapp]

TASK [metno.chrony : Check ntpd is running] ***************************************************************************************************************
ok: [snapp]

TASK [metno.chrony : Disable ntpd] ************************************************************************************************************************
skipping: [snapp]

TASK [metno.chrony : Chrony config] ***********************************************************************************************************************
changed: [snapp]

TASK [metno.chrony : Enable chrony service and start] *****************************************************************************************************
ok: [snapp]

TASK [metno.chrony : Restart chrony when config is changed] ***********************************************************************************************
changed: [snapp]

TASK [metno.chrony : Set timezone] ************************************************************************************************************************
changed: [snapp]

PLAY RECAP ************************************************************************************************************************************************
snapp                      : ok=9    changed=4    unreachable=0    failed=0    skipped=3    rescued=0    ignored=0
```

I ran it against a new Ubuntu Jammy virtual machine, running role version 3.2.0.

Which version of the role did you run with?

**Comment by KjetilKnutsen** [comment]
Correction, this is the same issue as #19. This will be fixed in the next version.

**Comment by KjetilKnutsen** [comment]
This has now been fixed in version 4.0.0. Closing issue.
